ggcnn_humanseg:
    camera_info:
        info: /camera/color/camera_info
        camera_frame: camera_color_optical_frame
        robot_base_frame: vx300s/base_link
        fov: 65.5
    camera:
        image: /camera/color/image_raw/compressed
        depth: /camera/aligned_depth_to_color/image_raw
    subscription:
        bodyparts: /bodyparts/topic
        egohands: /egohands/topic
        darknet: /darknet_ros/bounding_boxes
    interface:
        topic: /ggcnn_humanseg
    ggcnn:
        crop_size: 300
        crop_offset: 40
        out_size: 300
    robot:
        dist_ignore: 1
        grip_width: 0.08
    visualization: 
        topic: /ggcnn_humanseg/visualization
        activated: False
